src/Admin/BaseBlockAdmin.php

```diff
@@ -133,7 +154,14 @@ public function prePersist($object)
      */
     public function postPersist($object)
     {
-        $this->blockManager->get($object)->postPersist($object);
+        if (\is_callable([$object, 'postPersist'])) {
+            $this->blockManager->get($object)->postPersist($object);
+
+            @trigger_error(
+                'The '.__METHOD__.' method is deprecated since version 3.x and will be removed in sonata/block-bundle 4.0.',
```

Comment:
```suggestion
                'The '.__METHOD__.'() method is deprecated since sonata-project/block-bundle 3.x and will be removed in version 4.0.',
```